crates/red_knot_python_semantic/resources/mdtest/import/relative.md

```diff
@@ -28,14 +28,18 @@ reveal_type(X)  # revealed: Literal[42]
 
 ## Dotted
 
-```py path=package/__init__.py
+TODO: We currently have to artificially name this package to be unique within this mdtest file for
+tests to pass. Once we fix that issue, we should revert this back to `package` to be consistent with
+the rest of the file.
```

Comment:
This is no longer needed with https://github.com/astral-sh/ruff/pull/15030 merged.  Thanks @MichaReiser!